src/main/java/ca/corefacility/bioinformatics/irida/ria/web/launchPipeline/dtos/LaunchRequest.java

```diff
@@ -14,6 +17,45 @@
 	 */
 	private String description;
 
+	/**
+	 * List of file identifiers to run on the pipeline
+	 */
+	private List<Long> fileIds;
+
+	/**
+	 * Email the user when the pipeline is completed
+	 */
+	private boolean emailPipelineResult;
```

Comment:
Since the email pipeline result has changed from a checkbox to a select with 3 options this will also need to be updated accordingly

Reply:
I will make a new branch for this :(

Reply:
Sounds good. I'm sorry :(